Confuser.CLI/Confuser.CLI.csproj

```
@@ -1,118 +1,31 @@
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="..\GlobalAssemblyInfo.cs">
-      <Link>Properties\GlobalAssemblyInfo.cs</Link>
-    </Compile>
-    <Compile Include="Options.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Confuser.Core\Confuser.Core.csproj">
-      <Project>{BEB67A6E-4C54-4DE5-8C6B-2C12F44A7B92}</Project>
-      <Name>Confuser.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Confuser.DynCipher\Confuser.DynCipher.csproj">
-      <Project>{E832E9B8-2158-4FC0-89A1-56C6ECC10F6B}</Project>
-      <Name>Confuser.DynCipher</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Confuser.Protections\Confuser.Protections.csproj">
-      <Project>{3EAB01B5-9B49-48D8-BFA1-5493B26CCB71}</Project>
-      <Name>Confuser.Protections</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Confuser.Renamer\Confuser.Renamer.csproj">
-      <Project>{862DA0DA-52E1-47CD-B9C2-46B106031B28}</Project>
-      <Name>Confuser.Renamer</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Confuser.Runtime\Confuser.Runtime.csproj">
-      <Project>{A45C184F-F98F-4258-A928-BFF437034791}</Project>
-      <Name>Confuser.Runtime</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="..\ConfuserEx.snk">
-      <Link>Properties\ConfuserEx.snk</Link>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Properties\Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+  
+  <PropertyGroup Label="Assembly Information">
+    <Title>ConfuserEx Command-line</Title>
+    <Description>Command-line interface of ConfuserEx</Description>
+  </PropertyGroup>
+
+  <ItemGroup Label="Nuget Dependencies">
+    <PackageReference Include="NDesk.Options.Core" Version="1.2.5" />
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Nerdbank.GitVersioning">
-      <Version>3.3.37</Version>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
+
+  <ItemGroup Label="Project Dependencies">
+    <ProjectReference Include="..\Confuser.Core\Confuser.Core.csproj" />
+    <ProjectReference Include="..\Confuser.DynCipher\Confuser.DynCipher.csproj" />
+    <ProjectReference Include="..\Confuser.Protections\Confuser.Protections.csproj" />
+    <ProjectReference Include="..\Confuser.Renamer\Confuser.Renamer.csproj" />
+    <ProjectReference Include="..\Confuser.Runtime\Confuser.Runtime.csproj" />
```

It seems like only a single dependency is required since others are referenced indirectly anyway.

```suggestion
    <ProjectReference Include="..\Confuser.Protections\Confuser.Protections.csproj" />
```

---

Directly used projects should be directly referenced and not use recursive references to get all the dependencies needed. I'll reduce that to some extend, but `Core` will stay in because it contains the code that is actually used, and `Protections` as well as `Renamer` remains to ensure that all protections are available and will be deployed properly.